pandas-stubs/core/generic.pyi

```diff
@@ -94,7 +94,6 @@ class NDFrame(PandasObject, indexing.IndexingMixin):
     def bool(self) -> _bool: ...
     def __abs__(self) -> NDFrame: ...
     def __round__(self, decimals: int = ...) -> NDFrame: ...
-    def __iter__(self) -> Iterator: ...
```

Comment:
I wouldn't mind keeping this: there might be people inheriting from `NDFrame`.

Reply:
@Dr-Irv suggested removing this. Check [this comment](https://github.com/pandas-dev/pandas-stubs/issues/119#issuecomment-1179557772).

Reply:
> I wouldn't mind keeping this: there might be people inheriting from `NDFrame`.

`NDFrame` isn't in the public API, so people should _not_ be doing that!